egs/tunisian_msa/s5/local/chain/tuning/run_tdnn_1a.sh

```diff
@@ -0,0 +1,282 @@
+#!/bin/bash
+
```

Comment:
There should be results  here (and a compare_wer.sh script to compare them.).
@xiaohui-zhang, would you mind either running it yourself locally or asking one of the newer guys to do the same?  It would be good experience for them.

Reply:
sure I'll take care of it